app/lib/analytics.coffee

```diff
@@ -68,21 +64,23 @@ logToGoogle = (eventData) =>
 This will log a user interaction both in the Geordi analytics API and in Google Analytics.
 ###
 logEvent = (params) =>
-  eventData = buildEventData(params)
-  addUserDetailsToEventData(eventData)
-  .always (eventData) =>
-    logToGeordi eventData
-    logToGoogle eventData
+  return true
```

Comment:
Should we just remove this function rather than commenting it out?

Reply:
yeah - i don't think we'll run the experiment again so i'll remove all the code related to event logging, etc.